resources/qml/WelcomePages/UserAgreementContent.qml

```diff
@@ -27,39 +26,32 @@ Item
         renderType: Text.NativeRendering
     }
 
-    Item  // Area for pictures and texts
+    Label
     {
-        anchors.top: titleLabel.bottom
-        anchors.bottom: agreeButton.top
-        anchors.left: parent.left
-        anchors.right: parent.right
-        anchors.margins: UM.Theme.getSize("wide_margin").width
-
-        Label
+        id: disclaimerLineLabel
+        anchors
         {
-            id: disclaimerLineLabel
-            anchors.centerIn: parent
-            anchors.margins: UM.Theme.getSize("wide_margin").width
-
-            width: (parent.width * 2 / 3) | 0
-
-            text: "<p><b>Disclaimer by Ultimaker</b></p>"
-                + "<p>Please read this disclaimer carefully.</p>"
-                + "<p>Except when otherwise stated in writing, Ultimaker provides any Ultimaker software or third party software \"As is\" without warranty of any kind. The entire risk as to the quality and perfoemance of Ultimaker software is with you.</p>"
-                + "<p>Unless required by applicable law or agreed to in writing, in no event will Ultimaker be liable to you for damages, including any general, special, incidental, or consequential damages arising out of the use or inability to use any Ultimaker software or third party software.</p>"
-            textFormat: Text.RichText
-            wrapMode: Text.WordWrap
-            font: UM.Theme.getFont("default")
-            renderType: Text.NativeRendering
+            top: titleLabel.bottom
+            topMargin: UM.Theme.getSize("wide_margin").height
+            left: parent.left
+            right: parent.right
         }
+
+        text: "<p><b>Disclaimer by Ultimaker</b></p>"
+            + "<p>Please read this disclaimer carefully.</p>"
+            + "<p>Except when otherwise stated in writing, Ultimaker provides any Ultimaker software or third party software \"As is\" without warranty of any kind. The entire risk as to the quality and perfoemance of Ultimaker software is with you.</p>"
```

Comment:
This disclaimer has a typo: `perfoemance` -> `performance`. Looks like it's been there for a while. I'll fix that up.

Reply:
Fixed here: https://github.com/Ultimaker/Cura/pull/5449/commits/453a785beedbee7d43881e143bdc0400daa253c9